### ISSUE TITLE
Custom font doesn't load when PSD is uploaded. No warning of missing font

### ISSUE THREAD

**Issue by zoplonix** [photopea/photopea#977]
- I made a file with photopea
- Choose File > Open and opened a .ttf font
- Photopea said it was loaded
- I did my font work
- I saved as PSD
- I quit photopea
- I loaded photopea again
- I opened my PSD
- My custom font is gone, expecting to see my custom font or get warned that it was not loaded.

Workaround was to

- Choose File > Open and open my .ttf font
- Select each text that used my custom font, switch it to another font, then switch it back

I know fonts don't get stored in the PSD but photoshop will usually warn if it doesn't have a font in the PSD. And I think PSDs save prerendered vectors of all of the text that could be displayed instead of the current behaviour in photopea of swapping to a different font

**Comment by zoplonix** [comment]
What about a warning that the font isn't loaded like photoshop does? I found myself doing a whole bunch of work on text, trying to get it to fit in a certain area, and didn't notice the slight shift from the prerendered pixels to the default font. So it was quite annoying when all that work was for not and I had to do it all over again after I loaded the font. Also with multiple fonts on multiple layers your suggestion to edit each text to see what the font is and load it is extremely unreasonable when the software already knows it doesn't have the fonts and can simply notify the user
